doc/verification.rst

```diff
@@ -4,28 +4,75 @@ Verification
 Overview
 --------
 
-This is a SV/UVM testbench for the ibex core verification.
-This testbench loads the instruction binary generated from open source random instruction generator `riscv-dv <https://github.com/google/riscv-dv>`_, runs the RTL simulation, and compares the instruction trace against ISS simulation.
-
-Testbench component
-~~~~~~~~~~~~~~~~~~~
-
--  Random instruction generator : `RISCV-DV <https://github.com/google/riscv-dv>`_
--  `Memory interface agent for instruction fetch and load/store
-   operations <https://github.com/lowRISC/ibex/tree/master/dv/uvm/common/ibex_mem_intf_agent>`_
--  `Interrupt interface agent <https://github.com/lowRISC/ibex/tree/master/dv/uvm/common/irq_agent>`_
--  `Memory model <https://github.com/lowRISC/ibex/tree/master/dv/uvm/common/mem_model>`_
--  `Test and sequence library <https://github.com/lowRISC/ibex/tree/master/dv/uvm/tests>`_
+This is a SV/UVM testbench for verification of the Ibex core.
+At a high level, this testbench uses the open source `RISCV-DV random instruction generator
+<https://github.com/google/riscv-dv>`_ to generate compiled instruction binaries, loads them into a
+simple memory model, stimulates the Ibex core to run this program in memory, and then compares the
+core trace log against a golden model ISS trace log to check for correctness of execution.
```

Comment:
Should there be a description on the ISS itself (or at least point to a page that describes it)?

Reply:
Yep, there is a link to the Spike ISS github page in the 'Getting Started' section of this documentation